installation/linux.md

```diff
@@ -354,20 +354,77 @@ sudo rm -rf /opt/openhab2/
+
+* Package repository based installation:
+
+  ```shell
+  tail -f /var/log/openhab2/openhab.log -f /var/log/openhab2/events.log
+  ```
+
+* Manual installation:
+
+  ```shell
+  tail -f /opt/openhab2/userdata/logs/openhab.log -f /opt/openhab2/userdata/logs/events.log
+  ```
+
+You could even set up an SSH configuration (in Putty or similar) to automatically connect and execute the commands every time you start working on your setup.
+
+With openHAB 2 you can also [use the Karaf console]({{base}}/administration/logging.html#karaf-console) to have a colored glance at the logging information.
+
+## Recommended Additional Setup Steps
+
+The following is not directly related to the openHAB installation but rather recommended on a openHAB system.
+The need for these and the exact implementation on a specific system might differ from user to user.
+
+### Privileges for Common Peripherals
+
+An openHAB setup will often rely on hardware like a modem, tranceiver or adapter to interface with home automation hardware.
+Examples are a Z-Wave, Enocean or RXFcom USB Stick or a Raspberry Pi addon board connected to the serial port on its GPIOs.
+In order to allow openHAB to communicate with additional peripherals, it has to be added to corresponding Linux groups.
+The following example shows how to add Linux user `openhab` to the often needed groups `dialout` and `tty`.
+Additional groups may be needed, depending on your hardware and software setup.
 
-|                                  | Repository Installation  | Manual Installation (according to [guide](#manual-installation))  |
-|:--------------------------------:|:-------------------------|:------------------------------------------------------------------|
-| openHAB application              | `/usr/share/openhab2`    | `/opt/openhab2`                                                   |
-| Site configuration               | `/etc/openhab2`          | `/opt/openhab2/conf`                                              |
-| Log files                        | `/var/log/openhab2`      | `/opt/openhab2/userdata/logs`                                     |
-| Userdata like rrd4j databases    | `/var/lib/openhab2`      | `/opt/openhab2/userdata`                                          |
-| Service configuration            | `/etc/default/openhab2`  | (not preconfigured)                                               |
+```shell
+sudo useradd openhab dialout
+sudo useradd openhab tty
+```
 
-## Network Sharing
+Additionally it's needed to allow the java environment to access the serial port of the connected peripheral.
+Therefor the following setting has to be added/adapted on your system in either `/etc/default/openhab2` or `/usr/share/openhab2/runtime/bin/setenv`:
```

Comment:
Which one should this be? AFAIK, `/etc/default/openhab2` sets this parameter before setenv which appends to itself:

```
export EXTRA_JAVA_OPTS="${EXTRA_JAVA_OPTS_COMMON} ${EXTRA_JAVA_OPTS_ARCH} ${EXTRA_JAVA_OPTS}"
```

If this is the case, then it would be better to only suggest setting /etc/default/openhab2, especially since we want to give the impression that there should be nothing of interest to change in the application directory. At the moment, it may give users the impression that **overwriting** the current setenv export would work fine, but this is not the case.